src/components/account/selectedAccount.tsx

```
@@ -22,12 +21,19 @@ export const SelectedAccountInfo: React.FC<{}> = ({}) => {
-  } = context;
-
+  } = useSharedState();
   const { isCopyTooltipShown, showTooltip } = useCopyTooltip();
-
   const navigate = useNavigate();
 
+  useEffect(() => {
+    const accountConfig = async () => {
+      try {
+        await fetchAccountConfig();
+      } catch (error) {}
```

ESLint

[error] Empty block statement.

(no-empty)